resources/views/Auth/login.blade.php

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Custom Authentcation</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+          rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+          crossorigin="anonymous">
+</head>
+
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-md-4 col-md-offset-4" style="margin-top: 20px;">
+            <h4>Log in</h4>
+            <hr>
+            <form action="{{route('login-user')}}" method="post">
+                @csrf
+                @if(session('success'))
+                    <div class="alert alert-success">{{session('success')}}</div>
+                @endif
+                @if(session('fail'))
+                    <div class="alert alert-danger">{{session('fail')}}</div>
+                @endif
+                <div class="form-group">
+                    <label for="email">email adres</label>
```

Comment:
Hoofdletter in email adres.
Aangezien je wel een hoofdletter gebruikt voor het label Wachtwoord